### PR TITLE
Add help page for Python 3 issues

### PR DESCRIPTION
**Pull request by ibrahima**
Adds a little documentation about what to do if your autograder fails to start due to Python 3 issues.

### CODE REVIEW

File: docs/python3_issues.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+## Install Gradescope autograder harness dependencies
+
+This approach is a little more brittle, but you can also work around
+this issue by installing the dependencies our autograder harness needs
+within your custom Python version. The current list of dependencies is
+the following:
+
+```
+pyyaml pytz requests psutil grequests python-dateutil
```
Comment (SyntaxBlitz) [inline docs/python3_issues.md:48]: could we add a comment to `requirements.txt` or the top of `harness.py` reminding us to update these help docs if the harness dependencies change?

Comment (ibrahima) [reply docs/python3_issues.md:48]: Good idea! I'll add it somewhere.

**Review by mDibyo — APPROVED** [review]
Documentation looks good to me.

Could we resolve these issues by using a specific version of python3 for our harness instead, instead of relying on the default system version?

It sounds like that would make the harness a bit more robust, although long term, it sounds like the right thing to do would be to make the harness.py a standalone executable (via something like [Pyinstaller](https://pyinstaller.readthedocs.io/en/stable/operating-mode.html)).

**Comment by mDibyo** [comment]
(In any case, this looks good to merge for now.)

**Comment by ibrahima** [comment]
> Documentation looks good to me.
> 
> Could we resolve these issues by using a specific version of python3 for our harness, instead of relying on the default system version?
> 
> It sounds like that would make the harness a bit more robust, although long term, it sounds like the right thing to do would be to make the harness.py a standalone executable (via something like [Pyinstaller](https://pyinstaller.readthedocs.io/en/stable/operating-mode.html)).

The tricky thing about that is that our Fedora base image has different versions of Python than Ubuntu 18.04. There's a few ideas I have around this but unfortunately it's not as simple as just swapping python3 for python3.6.

**Pull request merged**